beetsplug/mpdstats.py

```diff
@@ -60,13 +55,8 @@ def __init__(self, log):
         self._log.debug('music_directory: {0}', self.music_directory)
         self._log.debug('strip_path: {0}', self.strip_path)
 
-        if sys.version_info < (3, 0):
-            # On Python 2, use_unicode will enable the utf-8 mode for
-            # python-mpd2
-            self.client = mpd.MPDClient(use_unicode=True)
-        else:
-            # On Python 3, python-mpd2 always uses Unicode
-            self.client = mpd.MPDClient()
+        # On Python 3, python-mpd2 always uses Unicode
```

Comment:
This comment is maybe not necessary anymore.

Reply:
Deleted